fuzz/src/full_stack.rs

```diff
@@ -1084,8 +1084,6 @@ fn two_peer_forwarding_seed() -> Vec<u8> {
 	// rest of open_channel and mac
 	ext_from_hex("030000000000000000000000000000000000000000000000000000000000000005 020900000000000000000000000000000000000000000000000000000000000000 01 0000 01021000 03000000000000000000000000000000", &mut test);
 
-	// One feerate request returning min feerate, which our open_channel also uses (ingested by FuzzEstimator)
-	ext_from_hex("00fd", &mut test);
```

Comment:
The high-level test summaries in `test_no_existing_test_breakage` and `test_gossip_exchange_breakage` both mention adding a new feerate request as test-breaking, that can be updated now 🎉